pkg/system/phase4_configuring.go

```diff
@@ -763,6 +763,20 @@ func (r *Reconciler) preparePVPoolBackingStore() error {
 	return nil
 }
 
+func (r *Reconciler) defaultBSCreationTimedout(minutesSinceCreation float64) bool {
```

Comment:
any reason not to pass the creation time and calculate `time.Since` inside this function?

Reply:
Yes, I'm using the `minutesSinceCreation` in the message. I can change the message and not pass this information and then change the code as you suggested.